bin/newman.js

```diff
@@ -96,6 +99,15 @@ program.on('command:*', (command) => {
     program.help();
 });
 
+program
+    .command('run-request <url>')
```

Comment:
This change does not seem atomic or aligned with the original theme of the PR

This also conflicts with the upcoming `Newman request` command.

I'm guessing we should remove this.

Reply:
@shamasis 
It crept in by mistake.
I was working on the request command as well.😅